grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -533,9 +533,14 @@ class UrlMappings {
             action = [POST: "validateInvoiceItem"]
         }
 
+        "/api/prepaymentInvoices/$id/invoiceItems" {
+            controller = "prepaymentInvoiceApi"
+            action = [POST: "updateItems"]
+        }
+
         "/api/prepaymentInvoiceItems/$id/" {
             controller = "prepaymentInvoiceItemApi"
-            action = [DELETE: "delete"]
+            action = [POST: "update", DELETE: "delete"]
```

Comment:
If I'm doing an update only on the provided fields I usually use PATCH over POST (I think that's the generally accepted convention) but I'm not really bothered either way.

You do have a trailing slash in the URI which is probably best to remove for consistency's sake but up to you.